[Blood DK] added DS timing graph

joshinat0r opened the pull request:
Adds a plot to the BDK analysis thats showing Death Strike casts and the HP.
Goal is to help spotting low HP situations where the BDK relied on external healing instead of DS-selfhealing.

![ds_timing](https://user-images.githubusercontent.com/29842841/37241530-d4d9f192-245a-11e8-88cb-dd06b7a449eb.PNG)

Gebuz commented:
Thanks for the PR! Would it be possible to get the DS dots match the height of current health like the purify ones do for monk? Makes it easier to see at what HP it was used. Also, since DK max HP fluctuates so much, would it make sense to show the health as percent of max health instead? If you think raw numbers are better I would suggest that you make the Max HP curve more obvious, it's a bit hard to see at the moment.

joshinat0r commented:
Updated the PR, showing the graph based on percent is actually better.
Displaying the DS dots at about 25% height of the graph was on purpose since the tooltip has more infos about the DS cast (how much it healed and overhealed for and at what HP), but hovering over the DS dot would show me the HP-graphs tooltip instead. Changed the tooltip now, so that shouldn't be an issue anymore.

![ds_timing2](https://user-images.githubusercontent.com/29842841/37244398-c6e3c26a-2488-11e8-9189-3fff405fd6d6.PNG)

Gebuz commented:
Oh I forgot to mention, remember to add a changelog entry in the future!

joshinat0r commented:
Should have read the guideline till the end :)
I'll add a changelog for this PR in my next PR.
